### DISCUSSION TITLE
Google will migrate old free G-suite accounts to paid plans. SimpleLogin for the rescue?

### DISCUSSION THREAD

**Discussion by dawidry** [simple-login/app#758] [Q&A] [answered]
Hi everyone,

I'm in the trap. I used G-suite for over decade as family mail solution. I have custom domain which we use.
Because not all family member want to pay for it, I'm looking for solution which give me possibility to keep all e-mail addresses working whatever email provider family members will choose.
Some of them want to "migrate" to free @gmail.com - but they would have ability to receive/send emails with custom address which they used for last 10 years. The other want to use @icloud.com (since they have Apple devices). So I started thinking about "forwarding service". SimpleLogin seems to be much more than just forwarding but for now that's all I need. I woud pay for Premium plan. I just need to cofirm and kindly ask you to comment if it will work:

1. Forwarding and receive emails send to:
member1@mydomain.com ---> member1@gmail.com
member2@mydomain.com ---> member2@icloud.com
member3@mydomain.com ---> member3@protonmail.com

2. Send emails from Gmail as member1@mydomain.com
3. Send emails from iCloud as member2@mydomain.com 
4. Plus "+" addressing support e.g.
I have defined alias member1@mydomain.com send to member1@gmail.com
Plus addressing means that:
member1+shopping@mydomain.com
member1+social@mydomain.com
should also go to member1@gmail.com

5. And all above with respect privacy. I read, SimpleLogin is open source which is huge advantage. But I don't see clear information if (and how) my data (in particular e-mail messages) are stored. If so, how long.

So to sum up: will SimpleLogin cover all my needs?

**Comment by trikotret** [comment, marked as answer]
Hi
I'll try to answer as much as I can. 
1. Each alias can have its own email forwarding address. So, your first question would work. This what's on their site about mailboxes **"A mailbox is just another personal email address. When creating a new alias, you could choose the mailbox that owns this alias, i.e:
- all emails sent to this alias will be forwarded to this mailbox
- from this mailbox, you can reply/send emails from the alias."**
2 & 3. In order to be able to send emails from your member1@mydomain.com, you would need to create a reverse alias for the recipient email It can easily be done on simplelogin site. For example, you want to email joe@gmail.com, you would create a reverse alias for joe. Joe will get the email from member1@mydomain.com. He would have no clue that it's from a personal gmail. To make things simple, as you make a reverse alias for an email, make a habit of saving the alias in your address book. This way each contact would have two emails saved. The main email and the reverse alias. So, when you decide to send that person an email, you'll see two drop downs to choose from. Always choose SL alias to show that email came from your domain. Hopefully SL will implement their own SMTP option down the road. 
4. You won't need to use + for aliases. Just make an alias and with catch all domain, you'll get that email. For example, I have an alias for netflix.me@domain.com.
5. Regarding privacy https://simplelogin.io/privacy/

**Reply by dawidry** [reply]
Thanks for answer. I appreciate it. 
4. One more regarding plus addressing. What I'd to achieve is to assure that if someone will send message such as member1+shopping@mydomian.com it will be delivered to member1 inbox (e.g. at gmail.com). But if other user (member2) used (in the past) plus addressing approach he/she sould get messages only those which are addressed to him/her (to member2).
Correct me if I'm wrong but with "catch-all" feature I will get all emails which use plus addressing (member1, member2 and so on)?
I don't have to have option to answer with this plus-addressing address. All what I need is to assure that it will be delivered but to different mailboxes.

5. Thanks, I read this Privacy page however it's not clear statement if emails are stored. And if so, how long? There is no clear information how SimpleLogin process with data which goes thru their servers. I'm not developer to inspect source code on github.

**Reply by trikotret** [reply]
Regarding 4. if you use catch all, it will go to main email. "Auto-created aliases are automatically owned by the following mailboxes" I think if you using one domain for two people, you would have to make aliases manually and apply a mailbox to it. Hopefully someone from SL can give more info. I've only been using SL for 2 months with two domains. One personal and other business.
5. Someone from SL would need to answer.
6. You can give SL a try for free and play around to get familiar with it.

**Reply by dawidry** [reply]
Thanks. Sure I'll give it try. I also consider forwardemail.net.


I'm also bit concern about reverse-alias mechanism. Sounds weird a bit. Creating revers-alias for each single person/recepiet? Hmm... I expected some solution to not have do things like that. Anyway thanks for explonation.

**Comment by r1ch** [comment]
@dawidry - In the same boat as you - and for any others that have the same question:

**Yes** you can solve this and it might be easier in future (https://app.simplelogin.io/dashboard/directory)

For the moment you can do this using auto-create rules:

Click on Domains > Details (for your domain) > auto-create rules

Now - _for every user_ - you need to create a rule:

`member1.*`  -> member1's account
`member2.*` -> member2's account

![image](https://user-images.githubusercontent.com/420973/150789451-41380228-2057-419a-9caf-a59e5aea7920.png)


This keeps things separate, ensures you get the dynamic creation you're used to with GSuite 😭 

(Strictly user+.* to keep to just + subaddressing)

**Reply by dawidry** [reply]
Looks promising. Thanks for advice.
